recipes/arpack/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "3.4.0" %}
+{% set variant = "openblas" %}
+
+package:
+  name: arpack
+  version: {{ version }}
+
+source:
+  fn: arpack-{{ version }}.tar.gz
+  url: https://github.com/opencollab/arpack-ng/archive/{{ version }}.tar.gz
```

Comment:
AFAICT you are correct to use this. Arch Linux and Gentoo both use this too. Not sure about other Linux distros though.